containers/config.ts

```diff
@@ -21,6 +21,7 @@ export enum NETWORK_NAMES {
   CRONOS = "Cronos",
   AURORA = "Aurora",
   METIS = "Metis",
+  MOONBEAM = "Moonbeam",
```

Comment:
This structure seems unnecessary. PFcore JSON output has user-visible strings per chain. 
![image](https://user-images.githubusercontent.com/90275425/150431213-02a87468-8bf9-4285-a1aa-cbafcb852902.png)